sql/parser/Makefile

```diff
@@ -0,0 +1,3 @@
+sql.go: sql.y
+	go tool yacc -o sql.go sql.y
+	@echo "$$(echo '// Code generated by go yacc.' | cat - sql.go)" > sql.go
```

Comment:
I don't think that `awk` command can work. Awk will begin outputting to the file and truncate the input. My outer echo was attempting to buffer the input, but it wasn't working properly. Switched to using a temporary file.